CMakeLists.txt

```diff
@@ -83,10 +83,23 @@ function(add_sample_app APP_NAME IDE_FOLDER SOURCE INCLUDE SHADERS ASSETS)
 
     if(PLATFORM_EMSCRIPTEN)
         set(RESOURCE_PATH "${PROJECT_SOURCE_DIR}/assets/")
-        target_link_options(${APP_NAME} PRIVATE "SHELL: -s ALLOW_MEMORY_GROWTH=1 --preload-file '${RESOURCE_PATH}@'")
+        target_link_options(${APP_NAME} PRIVATE "SHELL: -s ALLOW_MEMORY_GROWTH=1 --preload-file ${RESOURCE_PATH}@")
+
+        if (${APP_NAME} STREQUAL "GLTFViewer")
+            set(EXCLUDE_DIRECTORIES
+                ".git"
+                "BarbieDodgePickup"
+                "IridescenceAbalone"
```

Comment:
Let's remove IridescentDishWithOlives instead of IridescenceAbalone - it doesn't fully work in WebGPU as there is not enough textures

Reply:
Actually, we don't support Iridescence anyway, so it doesn't matter, but IridescenceAbalone just looks less broken than IridescentDishWithOlives